azure/durable_functions/models/entities/EntityState.py

```diff
@@ -0,0 +1,74 @@
+from typing import List, Optional, Dict, Any
+from .Signal import Signal
+from azure.functions._durable_functions import _serialize_custom_object
+from .OperationResult import OperationResult
+import json
+
+
+class EntityState:
```

Comment:
Not sure if EntityState is the best name for this, as it may be confused with the actual entity state that is a field on this object. Maybe `EntityBatchResult`?

Reply:
The name is actually copied from the JS implementation :p I was hoping to go through a refactoring of both repos at some point in the near future. There are also other confusing names that have been carried over from JS, so perhaps it makes sense to tackle them all in one go, but in a separate ticket, what do you think?

Reply:
That is fair... 

Blame 18 month ago Connor for that decision.

Reply:
And yes, I'm fine tracking this as a separate issue.

Reply:
Tracked: https://github.com/Azure/azure-functions-durable-python/issues/232